app/services/reject_applications_by_default.rb

```diff
@@ -3,6 +3,7 @@ class RejectApplicationsByDefault
   def call
     GetApplicationChoicesReadyToRejectByDefault.call.each do |application_choice|
       RejectApplicationByDefault.new(application_choice: application_choice).call
+      SendRejectByDefaultEmailToProvider.new(application_choice: application_choice).call
```

Comment:
What would you think of moving this into the `RejectApplicationByDefault` class?

Reply:
Yes, initially I added there, but I wasn't sure where should this responsibility live, I'm happy to refactor 👍